code/bit_manipulation/src/bit_division/bitdivision.cpp

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Please call the file the same thing as the folder.

Reply:
Done